lib/elixir_bench/github/client_http.ex

```diff
@@ -0,0 +1,45 @@
+defmodule ElixirBench.Github.ClientHTTP do
+
+  @behaviour ElixirBench.Github.Client
+
+  @params []
+  @base_url "https://raw.githubusercontent.com"
+  @ssl_options []
+
+  def get_plain(path) do
+    get(url(path), [{"accept", "text/plain"}], &(&1))
+  end
+
+  def get_json(path) do
+    get(url(path), [{"accept", "application/json"}], &Antidote.decode/1)
+  end
+
+  def get_yaml(path) do
+    get(url(path), [{"accept", "application/yaml"}], &decode_yaml/1)
+  end
+
+  @doc false
+  defp decode_yaml(content) do
+    case :yamerl.decode(content, [:str_node_as_binary, map_node_format: :map]) do
+      [parsed] -> {:ok, parsed}
+      _ -> {:error, :invalid}
+    end
+  end
+
+  defp get(url, headers, cb) do
+    options = [:with_body, ssl_options: @ssl_options]
+
+    case :hackney.get(url, headers, <<>>, options) do
+      {:ok, 200, _headers, data} ->
+        cb.(data)
+      {:ok, status, _headers, data} ->
+        {:error, {status, data}}
+      {:error, error} ->
+        {:error, error}
+    end
+  end
+
+  defp url(path) do
+    Path.join(@base_url, path) <> "?" <> URI.encode_query(@params)
```

Comment:
we seem to have totally given up on the ability to pass parameters along?

It seems like we didn't need it before so we could drop it imo until we need it again, but then we can also drop this whole part and just leave it at `Path.Join`.

I have a feeling they might crop up/be needed again eventually but we could wait until that happens